tests/unit/conftest.py

```diff
@@ -122,6 +122,21 @@ def db_name() -> str:
     return "database"
 
 
+@pytest.fixture
+def mock_db_name() -> str:
+    return "mock_database"
+
+
+@pytest.fixture
+def db_description() -> str:
+    return "database description"
+
+
+@pytest.fixture
+def mock_db_description() -> str:
+    return "mock database description"
+
```

Comment:
What's the difference between `db_name`, `db_description` and their `mock` counterparts?

Reply:
db_name (and db_description) fixtures are used everywhere, which leads to cases, when the parameter passing is not being checked correctly. For example in tests/unit/service/test_database.py:33, previously if we would pass something different to the `database = manager.databases.create(name="other_db_name")`, the next check `assert database.name == db_name` would still work correctly, which is a wrong behaviour.
In my opinion, all listed fixtures are unnecessary, however removing them would lead to a lot of refactoring.

Reply:
I'm not sure how changing the name in the mock fixes the issue you're outlining. In test_database.py:33 when a new name is passed to `manager.databases.create` nothing changes since httpx still has the databases url callback mocked to return the result with `db_name`, which passes the assert below. If that's modified to use `mock_db_name` the issue still stands e.g. `manager.databases.create(name="yet_another_db_name")` would pass the `assert database.name == "other_db_name"`.
I think in order to fix this and avoid any mistakes made in testing we can check what's being sent to the mock API in the test itself. e.g. `httpx_mock.get_request( <insert_expected_request_here> )`

Reply:
After decoupling the parameters, the `test_databases_create` started to fail, showing us that, there is an issue. Having just one `db_name`, that we use everywhere obfuscates such problems.

By splitting `db_name` and `mock_db_name` I didn't fix the issue. The issue was fixed by extending the `create_databases_callback`.

Reply:
This seems like we're testing the fixture and not the sdk itself.
My suggestion would be to remove the mock_database fixture from the callback function and create another database object inside a callback with provided parameters, and then return it's `dict()`. This way it seems to be simpler and representative.
Depending on different database names seems too implicit and not very obvious

Reply:
As suggested by @stepansergeevitch, I would remove both use `mock_database` as a function, then the `mock_db_name` `mock_db_description` would not be needed anymore, and I will remove them as well. @ptiurin are you fine with this approach?

Reply:
Sounds good.

Reply:
I figured, that `mock_database` is used much more, then I thought, so this solution is also not optimal